internal/lsp/rego/rego.go

```diff
@@ -78,6 +92,46 @@ func AllBuiltinCalls(module *ast.Module) []BuiltInCall {
 	return builtinCalls
 }
 
+//nolint:gochecknoglobals
+var regalRules = func() bundle.Bundle {
+	regalRules := rio.MustLoadRegalBundleFS(rbundle.Bundle)
+
+	return regalRules
+}()
+
+// AllKeywords returns all keywords in the module.
+func AllKeywords(ctx context.Context, module *ast.Module) (map[string][]KeywordUse, error) {
```

Comment:
This is currently pretty slow for ast.rego so I'm working on it's performance.

Reply:
I have this down to around 400ms on the first run, and then <50ms on the later runs. This is still not ideal, but a lot better than where it started at ~3s

Reply:
c92431ecc20054f6d646fa84b3f697f021b8ff3f
contains the improvements